draft-ietf-quic-transport.md

```diff
@@ -1487,8 +1487,8 @@ network.  This section describes the protocol for migrating a connection to a
 new client address.
 
 Migrating a connection to a new server address is left for future work. If a
-client receives packets from a new server address that are decryptable, the
-client MAY discard these packets.
+client receives packets in the connection from a new server address, the client
```

Comment:
nit: I don't think "in the connection" is necessary, since you begin the sentence with "Migrating a connection..."